facades/cmc_facade.py

```diff
@@ -0,0 +1,26 @@
+import requests
+import locale
+import json
+import os
+from dotenv import load_dotenv
+
+load_dotenv()
+
+def get_btc_price():
+  url = 'https://pro-api.coinmarketcap.com/v2/cryptocurrency/quotes/latest'
+
+  parameters = {
+    'symbol': 'BTC'
+  }
+
+  headers = {
+    'Accepts': 'application/json',
+    'X-CMC_PRO_API_KEY': os.getenv('CMC_API_KEY')
+  }
+  try: 
+    response = requests.get(url, params=parameters, headers=headers)
+    response.raise_for_status() # Raise an HTTPError for bad responses
+    price = response.json()['data']['BTC'][0]['quote']['USD']['price']
```

Comment:
consider adding some data validation so we don't end up with `KeyError` if we experience and issue with the API
```python
data = response.json()
if 'data' not in data or 'BTC' not in data['data']:
    raise ValueError("Unable to validate response structure")
```

Reply:
you could also catch key or value errors in an exception too.